qemu/tests/block_resize.py

```diff
@@ -158,6 +158,10 @@ def get_md5_of_file(filename):
             error_context.context("Extend disk to %s in guest"
                                   % block_size, logging.info)
             if os_type == 'windows':
+                if int(block_size) >= 2199023255552 and 'bios' in params['_short_name_map_file']['guest-hw.cfg']:
```

Comment:
use static var better than number directly. for example
max_bios_block_size=2199023255552 
if int(block_size) >=max_bios_block_size